asset-management-labs/automated-curation.ipynb

```diff
@@ -666,7 +666,11 @@
    "source": [
     "oakDeneLandingAreaFullPath = \"/\" + landingAreaDirectoryName + \"/oak-dene/clinical-trials/drop-foot\"\n",
     "\n",
-    "os.listdir (oakDeneLandingAreaFullPath)"
```

Comment:
In future (a later change?) we could use `kubectl exec lab-odpi-egeria-lab-core-0 -- ls /deployments`
But
 - need to know if using k8s or not
 - kubectl isn't currently in image
 - we could use python (/run/secrets/kubernetes.io contains credentials for the serviceaccount)
 
 Or could setup SSH
 
 But none of this works today.

Reply:
I added the one line update again for xtdb:
xtdbRepositoryOption     = "xtdb-inmemory"